crypto/constant_time_test.c

```diff
@@ -42,6 +42,9 @@
  * copied and put under another distribution licence
  * [including the GNU Public Licence.]
  */
+#ifdef __cplusplus
+extern "C" {
+#endif
```

Comment:
If this change is important then please make it in a separate PR that explains why it was done. It shouldn't be needed, AFAICT.